app/main/lib/similarity.py

```diff
@@ -80,8 +90,9 @@ def model_response_package(item, command):
 
 def add_item(item, similarity_type):
```

Comment:
Feels like we could have an optional callback URL here rather than assuming the callback is always CheckAPI. We'd then store that callback in Reddis with a unique id before sending the request to Presto (with that unique id)